Dockerfile

```diff
@@ -23,29 +23,40 @@
 # Determine final build variant [prod | test].
 ARG variant
 
-FROM golang:1.16.7 AS base
+# Base image
+FROM golang:1.17-buster AS base
+
 # Transfer all project files to container.
 WORKDIR /go/src/app
 COPY . .
+
 # Build and export cip command.
 RUN ./go_with_version.sh build ./cmd/cip
 RUN cp ./cip /bin/cip
+
 # Provide docker config for repo information.
 RUN mkdir /.docker
 RUN cp ./docker/config.json /.docker/config.json
 
+# gcloud-base image
 FROM gcr.io/google.com/cloudsdktool/cloud-sdk:latest AS gcloud-base
```

Comment:
Depending on what exactly kpromo uses from the sdk, it may be better if we use the `:slim` version as it it is half the size. I'm starting to dive into the code to see if we can - better yet- get rid of the calls to external binaries.